garage/tf/algos/ddpg.py

```diff
@@ -47,7 +47,7 @@ def __init__(self,
         Construct class.
 
         Args:
-            env(): Environment.
+            env_spec(): Environment specification.
```

Comment:
does't this have a type?

Reply:
It is `akro.Space`?

Reply:
uhh i think it's https://github.com/rlworkgroup/garage/blob/master/garage/envs/env_spec.py for now.